src/htmx.js

```diff
@@ -75,6 +75,7 @@ return (function () {
                 globalViewTransitions: false,
                 methodsThatUseUrlParams: ["get"],
                 selfRequestsOnly: false,
+                ignoreTitle: false,
```

Comment:
This is a source change, was that a mistake?

Reply:
@alexpetros It's a configuration option that was mentioned in the doc, and used in the code, but not defined in the config object
> 1 property was missing from htmx.config (ignoreTitle)

This option was introduced by c230931 which also [added documentation](https://github.com/bigskysoftware/htmx/commit/c230931d42489a629d685bedb440fadd7eaf1416#diff-e772080787482bb78eeddd4d3c75a4ecd414c5e06674fde21d4383c1c3927735R1635) for it, but it seems it was forgotten from the htmx.config object iself
That documentation itself was saying:
> defaults to `false`, if set to `true` htmx will not update the title of the document when a `title` tag is found in new content

So I added that default value of false in the config

Reply:
It's still gotta be in a separate PR. I know that seems picky but it's for a good reason. If you pull it out I can merge this one.

Reply:
Oh ok let me split that then